README.md

```diff
@@ -1,2 +1,22 @@
 # hw01
-Repository for homework 1
+
+Rachel Cundiff's Biography
+--------------------------
+
+I'm Rachel, a native Chicagoan in the MAPSS program here at UChicago concentrating in psychology/neuroscience. My thesis examines how certain psychological conditions, specifically depression and alexithymia, affect a patient's response to treatment for chronic pain. I have a beautiful kitten named Zelda who loves to run around my tiny apartment on the north side of the city. Here's a picture of her. I mean, just look at how cute she is!! 
+
+![Alt text][image]
+
+[image]: image/Zelda4.JPG
+
+Here's a list of my favorite TV shows:
+
++ Peaky Blinders
++ Man in the High Castle
++ The Dick van Dyke Show
++ The Crown
++ Law & Order: SVU
+
+I went to Loyola University Chicago for undergrad, and I'm really excited that our men's basketball team is in the final four for the NCAA tournament. **GO RAMBLERS!!**
+
+Finally, if you're interested, here is [my professional website!](https://rcundiff.github.io/)
```

Comment:
I didn't know you could add a website link! Very creative idea, and good to know. Your readme was very easy to follow, and your coding looks great.